packages/compiler/src/lib/visibility.ts

```diff
@@ -180,6 +181,9 @@ export const $parameterVisibility: ParameterVisibilityDecorator = (
 /**
  * Returns the visibilities of the parameters of the given operation, if provided with `@parameterVisibility`.
  *
+ * @deprecated Use `getParameterVisibilityFilter` instead.
+ *
+ * @see {@link getParameterVisibilityFilter}
```

Comment:
The two deprecations on the operation visibility decorators will have to be approved by the breaking change review board.  I will be surprised if anyone is using these filters, other than possibly the typespec-autorest emitter.

Reply:
There are no occurrences of `getReturnTypeVisibility` or `getParameterVisibility` within the `packages` folder of typespec-azure. I doubt anything other than typespec itself was using these APIs.

Reply:
Breaking change was approved.